airbyte-webapp/src/views/Connector/ConnectorCard/ConnectorCard.tsx

```diff
@@ -89,9 +134,8 @@ export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEdi
 
   const job = jobInfo || LogsRequestError.extractJobInfo(errorStatusRequest);
 
-  const { selectedConnectorDefinitionSpecification, onServiceSelect, availableServices, isEditMode } = props;
-  const selectedConnectorDefinitionSpecificationId =
-    selectedConnectorDefinitionSpecification && ConnectorSpecification.id(selectedConnectorDefinitionSpecification);
+  // Fill form with existing connector values otherwise set the default service name
+  const formValues = isEditMode ? props.connector : { name: selectedService?.name };
```

Comment:
I'd suggest we move the default name logic into the `ServiceForm` where the initialValues of the Formik context are actually calculated, so that we have all the "formik" values logic ideally inside the `ServiceForm` as much as possible and not passing those values from the outside.

Reply:
Sounds reasonable, and I also did a try:
The first thing which confuses me, we just replace one prop(formValues) with another dependency - connector.
And here is the place where the errors come:
<img width="664" alt="Screenshot at Oct 28 01-20-44" src="https://user-images.githubusercontent.com/20929344/198417484-047d680d-ce7e-424b-9a9c-e7a38c645565.png">
Because of:
```
interface ConnectorCardCreateProps extends ConnectorCardBaseProps {
  isEditMode?: false;
}

interface ConnectorCardEditProps extends ConnectorCardBaseProps {
  isEditMode: true;
  connector: ConnectorT;
}

export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEditProps>
```
Also did a try to pass the `connector` prop implicitly to `<ServiceForm />` just for testing, and surprisingly, the tests start failing...

Should I continue digging into it? 
Since the PR already has some changes out of the scope.